test/run_test.py

```diff
@@ -1688,7 +1677,7 @@ def __str__(self):
 
     if not options.no_translation_validation:
         os.environ["PYTORCH_TEST_WITH_TV"] = "1"
-
+    return
```

Comment:
This looks like either a copy/paste mistake or a left-over debug `return` statement

Reply:
Gonna leave this in until right before I actually merge